code/modules/mob/mob.dm

```diff
@@ -657,7 +664,7 @@ var/next_mob_id = 0
 			fall()
 		else if(ko || (!has_legs && !ignore_legs) || chokehold)
 			fall(forced = 1)
-	canmove = !(ko || resting || stunned || chokehold || buckled || (!has_legs && !ignore_legs && !has_arms))
+	canmove = !((is_nearcrit(src) ? 0 : ko) || resting || stunned || chokehold || buckled || (!has_legs && !ignore_legs && !has_arms))
```

Comment:
Yes but the issue is `ko` is made up of `weakened || paralysis || stat || (status_flags & FAKEDEATH)`

As it is now it seems like going from health>0 to -50<health<0 means that even if you were unconscious, paralysed or weakened then you can still move...